bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceOverlay.java

```diff
@@ -629,17 +629,17 @@ private void createReplaceToggle() {
 		GridDataFactory.fillDefaults().grab(false, true).align(GridData.BEGINNING, GridData.FILL)
 				.applyTo(replaceToggle);
 		replaceToggle.setToolTipText(FindReplaceMessages.FindReplaceOverlay_replaceToggle_toolTip);
-		replaceToggle.setImage(FindReplaceOverlayImages.get(FindReplaceOverlayImages.OBJ_OPEN_REPLACE_AREA));
+		replaceToggle.setImage(FindReplaceOverlayImages.get(FindReplaceOverlayImages.KEY_OPEN_REPLACE_AREA));
```

Comment:
```suggestion
		replaceToggle.setImage(FindReplaceOverlayImages.get(FindReplaceOverlayImages.OPEN_REPLACE_AREA));
```

What do you think of removing the useless prefixes? This is a perfect opportunity to test the Find/Replace overlay 😉

Reply:
You mean the "KEY" prefix?

Reply:
@BeckerWdf yes! Back when I had icons in LCL and in OBJ, the prefixes made (a bit of) sense, but now that everything is LCL, the prefix does not help in understanding the variables

Reply:
I added the "KEY" by intention to make it clear that this is not the filename

Reply:
@BeckerWdf Okay, then leave it in!